data/Remnant Cognizance.txt

```diff
@@ -0,0 +1,626 @@
+	passengers 1
+	source "Aventine"
+	destination "Esquiline"
+	to offer
+		has "Remnant: Cognizance 7: done"
+	on offer
+		log "People" "Gavriil" `A technician that works with Plume, and a talented astrobiologist. Gavriil is the project lead for the repurposed esquiline station.`
+		conversation
+			`You walk into Plume's lab to be greeted by a large stack of crates and equipment. Plume and another remnant are gathered around the stack checking items off a list. "Ah, <first>! Good timing. We were just finishing the inventory of supplies that need to go on this delivery. He gestures to the engineer, "this is Gavriil, one of our best technologists and a skilled astrobiologist in his own right. He will be going with you."`
+			`	"It will be a pleasure to work with you" sings Gavrill, making a short bow. "We will be traveling to a research station on Esquiline. Yes, it is a gas-world." He smiles at that. "I have to get some equipment setup and get the core systems online. There will be a few more loads of materials afterwards." He looks askance at you. "Shall we get started?"`
+				accept
+	on complete
+		payment 100000
+		conversation
+			`Gavriil gives you precise directions to a co-ordinate that your sensors tell you is an empty area, and when you reach the designated spot, you see a small station shimmering out of the clouds. "We never managed to actually cloak the station," grins the engineer. "But we managed to make it hide very effectively against every form of scanning we knew of. We even managed to give it a cloak-like field that renders it mostly transparent."`
+			`	He nods approvingly. "Rather impressive what my predecessors managed to do with what they had back then."`
+			choice
+				`	"What do you mean?"`
+				`	"Let's get on with it."`
+					goto skipinfo
+			`	"Well, this place is fairly old. It was built shortly after we learned how to build Starlings, and we had trouble extending the cloaking field beyond small ships. I don't know the mechanics of it, but it has taken close to two centuries of study and development for them to get cloaks to the point where they are feasible on large ships." We had plenty of gascraft back then, and the techniques for building high-pressure systems was well understood. The Alphas hadn't shown any interest in gas giants or 'gaslined ships' as we call them, so having somewhere like this to hide in makes sense. Best place to hide is somewhere no-one will look, right?." With that he nods to himself, climbs to his feet, and heads towards the suit locker.`
+			label skipinfo
+			`	The station should be pressurized to normal atmosphere, but it has been running on standby for ages with just yearly maintnance. It is time to get this place up and running again." With two of you running the equipment, everything is unloaded in short order. You notice that the last crate includes a substantial amount of field rations and survival gear. It looks like he is planning on living rough for a while.`
```

Comment:
`	"The station should be pressurized with a normal atmosphere, but it has been running on standby for ages with just yearly maintenance. It is time to get this place up and running again." With two of you running the equipment, everything is unloaded in short order. You notice that the last crate includes a substantial amount of field rations and survival gear. It looks like he is planning on living rough for a while.`

Reply:
Fixed.